backend/src/common/services/gift.ts

```diff
@@ -1,13 +1,49 @@
 import { KnownBlock } from '@slack/client'
 import _range from 'lodash/range'
 import '../../models/gift.model'
-import Gift, { IGiftDocument } from '../../models/gift.model'
+import Gift, { IGift, IGiftDocument } from '../../models/gift.model'
 import SlackConsts from '../consts/slack'
 import TranslationsService from './translationsService'
 
 export default class GiftService {
   private translationsService: TranslationsService
-
+  private initGifts: IGift[] = [
```

Comment:
initial gifts list should be stored in a separated file, instead of gift service